app/projectsmodel.cpp

```diff
@@ -48,14 +48,18 @@ void ProjectModel::findProjectFiles()
     projectFile.projectName = project.projectName;
     projectFile.projectNamespace = project.projectNamespace;
     QDateTime created = fi.created().toUTC();   // TODO: why UTC ???
-    if ( !project.qgisProjectFilePath.isEmpty() )
```

Comment:
I was thinking if LocalProjectInfo has new function `setInvalid(const std::string& reason)` or something similar, we can pass the error reason this way. this implementation does not set isValid == false, but the project is invalid. Also it uses projectFilePath as way how to pass the error around, which would not be required...

Reply:
Good suggestion - I am easy, also happy to merge it as is... your call Peter - change it?

Reply:
if it is not too much struggle i would change it, my concern is that isValid() function with the current implementation does not say truth. and Tomas wants to hear the truth...